### DISCUSSION TITLE
Install flaresolver in custom docker that uses ubuntu:22.04

### DISCUSSION THREAD

**Discussion by pbrns** [FlareSolverr/FlareSolverr#1389] [Q&A]
Hi to all, I need some advice and help in something. 
I want to host the flaresolverr inside docker but not in the simple image of python that has. I want to have a docker ubuntu to run it.

I do a lot of tests and configurations, after many days, I manage to start the flaresolverr but the only thing that I am seeing is that
```
2024-10-15 18:45:02 INFO     FlareSolverr 3.3.21
2024-10-15 18:45:02 INFO     Testing web browser installation...
2024-10-15 18:45:02 INFO     Platform: Linux-6.10.4-linuxkit-x86_64-with-glibc2.35
2024-10-15 18:45:02 INFO     Chrome / Chromium path: /usr/bin/google-chrome
2024-10-15 18:45:02 INFO     Chrome / Chromium major version: 129
2024-10-15 18:45:02 INFO     Launching web browser...
```

Because in the same docker I am running some other services, I don't want to have the simple docker image for flaresolverr that is the python build env.
This is my dockerfile (that I try to use).
[dockerfile.txt](https://github.com/user-attachments/files/17381349/dockerfile.txt)

it works as far as I try to run the flaresolverr.
It seems that the issue on the beginning was that chromium not exist. I am wondering how it works in the docker image of flaresolverr and not in the ubuntu image.
Python version is 3.10

I appreciate any help I can get.

Thanks in advance.

**Comment by ilike2burnthing** [comment]
Current version requires Python v3.11, however the issue you're running into is likely https://github.com/FlareSolverr/FlareSolverr/issues/1318, so you'll need to use https://github.com/21hsmw/FlareSolverr/tree/nodriver-support.
